### DISCUSSION TITLE
Radio button label position  v7.5.0

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#5687] [Q&A]
I think `Radio` label position nor works on v7.5.0. 
is this my browser problem? I'm using Arc browser

https://github.com/mantinedev/mantine/assets/66766189/dbc51db0-55bb-44fa-8108-645a47488a20

**Comment by rtivital** [comment]
It is a known issue, the fix will be available with the next patch.

**Comment by comertcimen** [comment]
It is fixed here. #5648
